hybrasyl/Dialogs.cs

```diff
@@ -302,6 +311,14 @@ public override void ShowTo(User invoker, VisibleObject invokee)
                 // Depending on how this was registered, it may not have an associate; thankfully we always
                 // get a hint of one from the 0x3A packet
                 var associate = Sequence?.Associate is null ? invokee : Sequence.Associate;
+                if (NextSequence.ToLower().Trim() == "mainmenu")

```

Comment:
Is there a way that these can be defined in an `enum` rather than `string` values?

Reply:
This is a value defined by scripting, it's easier to compare a string. I'll make this an `Enum` if it ever has more than one value defined.